bootstrap/kubeadm/config/manager/manager.yaml

```
@@ -22,6 +22,7 @@ spec:
         - "--leader-elect"
         - "--metrics-bind-addr=localhost:8080"
         - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}"
+        - "--bootstrap-token-ttl=${BOOTSTRAP_TOKEN_TTL:=0}"
```

```suggestion
        - "--bootstrap-token-ttl=${KUBEADM_BOOTSTRAP_TOKEN_TTL:=0}"
```
should we namespace the env var?

---

Sounds good.